portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class Comment  {
+
+  private final long id;
```

Comment:
does gson process private fields?  That could be your issue.  You might want to add public getters?